code_examples/core_features/did/03_light_did_migrate.ts

```diff
@@ -17,10 +19,13 @@ export async function migrateLightDid(
 
   // The new information is fetched from the blockchain and returned
   const migratedFullDidUri = Kilt.Did.getFullDidUri(lightDid.uri)
-  const migratedFullDid = await Kilt.Did.fetch(migratedFullDidUri)
+  const encodedUpdatedDidDetails = await api.call.did.query(
+    Kilt.Did.toChain(migratedFullDidUri)
+  )
+  const { document } = Kilt.Did.linkedInfoFromChain(encodedUpdatedDidDetails)
```

Comment:
I think we can finally do it like this everywhere
```suggestion
  return Kilt.Did.linkedInfoFromChain(encodedUpdatedDidDetails).document
```

Reply:
https://github.com/KILTprotocol/docs/pull/168/commits/bde139300ee59de5a225c8ed9307d9ce59936c00